### PR TITLE
Docs/daily workflow

### PR DESCRIPTION
**Pull request by tomchentw**
## 📝 Description

Runs these two scripts daily on GitHub Actions and commit the results to the repository:

* `yarn avatars:gen`
* `yarn members:gen`

## ⛳️ Current behavior (updates)

These scripts needs to be run by parts of the build process.

## 🚀 New behavior

Runs these two scripts daily on GitHub Actions. This saves time for local development as well as reducing the API usage.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,9 +58,9 @@
     "release": "changeset publish",
     "docs": "yarn workspace @chakra-ui/docs",
     "docs:deploy": "yarn build && yarn docs:build-ci && cp -r website/public .",
-    "docs:build-ci": "yarn members:gen && yarn docs:build",
-    "docs:build": "yarn avatars:gen && yarn search-meta:gen && yarn docs build",
-    "docs:dev": "yarn avatars:gen && yarn docs dev",
+    "docs:build-ci": "yarn docs:build",
```
Comment (with-heart) [inline package.json:61]: Could you remove the `docs:build-ci` script? We can just use `yarn docs:build` in `docs:deploy` now.

**Review by with-heart — APPROVED** [review]
This is a nice little improvement to our workflows. I like the idea of removing the gen commands from the docs build itself as it's not really a direct concern.

**Comment by with-heart** [comment]
@allcontributors please add @tomchentw for infra

File: package.json (hunk 1 above)
Comment (tomchentw) [reply package.json]: @with-heart now removed

**Comment by with-heart** [comment]
Thank you!

**Pull request merged**